yaml/xpu_functions.yaml

```diff
@@ -36,6 +36,9 @@ supported:
   - remainder_.Tensor
   - remainder.Tensor_out
   - remainder.Scalar_Tensor
+  - xlogy.Tensor
```

Comment:
Please remove these lines and add schemas in `native_functions.yaml` only.

Reply:
Done.